demos/signature/EVP_Signature_demo.c

```diff
@@ -0,0 +1,235 @@
+/*-
+ * Copyright 2021 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+/* 
+ * An example that uses the EVP_MD*, EVP_DigestSign* and EVP_DigestVerify* 
+ * methods to calculate and verify a signature of two static buffers.
+ */
+
+#include <string.h>
+#include <stdio.h>
+#include <openssl/err.h>
+#include <openssl/evp.h>
+#include "EVP_Signature_demo.h"
+#include <openssl/decoder.h>
```

Comment:
Another small NIT: Move #include <openssl/decoder.h> up one line above "EVP_signature_demo.h"

Reply:
Done in latest upload, thanks for your comments.